docs/themes.md

```diff
@@ -4,10 +4,12 @@
 zcli themes commands helps with managing Zendesk Help Center theming workflow.
 
 * [`zcli themes:preview [THEMEDIRECTORY]`](#zcli-themespreview-themedirectory)
+* [`zcli themes:import [THEMEDIRECTORY]`](#zcli-themesimport-themedirectory)
+* [`zcli themes:publish`](#zcli-themespublish)
```

Comment:
nit: should `zcli themes:publish` also accept a `[THEMEDIRECTORY]` for consistency?

Reply:
Wait, sorry. I think I get it now. We don't need the `[THEMEDIRECTORY]` because we are not performing an action on the local theme. We are publishing a theme that is already in the account. Sorry for the confusion 🤦